packages/wrangler/src/parse.ts

```diff
@@ -139,6 +143,25 @@ export function parseJSON<T>(input: string, file?: string): T {
 	}
 }
 
+/**
+ * A wrapper around `JSONC.parse` that throws a `ParseError`.
+ */
+export function parseJSONC<T>(input: string, file?: string): T {
+	const errors: JsoncParseError[] = [];
+	const data = jsoncParse(input, errors);
+	if (errors.length) {
+		console.log(JSON.stringify(errors));
+		throw new ParseError({
+			text: printParseErrorCode(errors[0].error),
+			location: {
+				...indexLocation({ file, fileText: input }, errors[0].offset + 1),
+				length: errors[0].length,
+			},
+		});
+	}
+	return data;
+}
+
```

Comment:
Is this being tested with unit or integrations tests?

Reply:
...it is now 😅 I've added unit tests for parsing JSONC to `parse.test.ts`

Reply:
Thanks!